control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+if ! hash virtualenvwrapper.sh 2>/dev/null; then
+    echo "Please install virtualenvwrapper and make sure it is in your PATH"
+    echo ""
+    echo "  sudo apt-get install git python-dev python-pip"
+    echo "  sudo pip install virtualenv virtualenvwrapper"
+    return 1
+fi
+
+source virtualenvwrapper.sh
+
+if [ ! -d ~/.virtualenvs/ansible ]; then
+    echo "Creating ansible virtualenv..."
+    mkvirtualenv ansible
+    pip install -r ~/commcarehq-ansible/ansible/requirements.txt
+else
+    workon ansible
+fi
+
+if ! [ -L ~/commcarehq-ansible/config -o -d ~/commcarehq-ansible/config ]; then
+    # TODO setup permissions (add user to group?) to allow write to config repo
+    #su ansible -c "usermod -a -G ansible $USER"
+    ln -s /home/ansible/commcarehq-ansible/config ~/commcarehq-ansible/config
```

Comment:
This is still shared, so still has most of the same problems we had before: people stepping on other people's work or accidentally leaving uncommitted changes. I still need to figure out how commits to this repo will workr. One advantage of committing from separate dev accounts is that each dev will have a separate git config, so commits should show correct author details.